basis/locals/parser/parser-tests.factor

```diff
@@ -67,3 +55,25 @@ IN: locals.parser.tests
     ] with-compilation-unit
     [ locals>> [ name>> ] map ] [ keys ] bi*
 ] unit-test
+
+<<
```

Comment:
I'm fairly sure it's needed. `<<` and `>>` does some kind of magic so you need them when running tests that tests the parser.